docs/cloud/alerts-configuration-manager.md

```diff
@@ -0,0 +1,71 @@
+# Creating Alerts with Netdata Alerts Configuration Manager
+
+The Netdata Alerts Configuration Manager enables users to create alerts from the Netdata Dashboard with an intuitive user interface.
```

Comment:
@sashwathn should we mention this feature is ONLY available for Netdata business users?

Reply:
> @sashwathn should we mention this feature is ONLY available for Netdata business users?

Should we include this in the documentation? No hard adding it I guess..

Reply:
Done..